source/includes/fundamentals/code-snippets/tls.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"os"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+
+	caFile := "<path to CA certificate>"
+	certFile := "<path to public client certificate>"
+	keyFile := "<path to private client key>"
+
+	// Load CA certificate file
+	caCert, err := os.ReadFile(caFile)
+	if err != nil {
+		panic(err)
+	}
+	caCertPool := x509.NewCertPool()
+	caCertPool.AppendCertsFromPEM(caCert)
```

Comment:
Optional: Check the "ok" bool returned by `AppendCertsFromPEM` to make sure it actually did something. Panic with a message if it failed.

E.g.
```go
if ok := caCertPool.AppendCertsFromPEM(caCert); !ok {
	panic("expected CA certificates file to be PEM format")
}
```